metricbeat/docs/modules/kubernetes.asciidoc

```diff
@@ -102,6 +102,10 @@ rules:
   - events
   - pods
   verbs: ["get", "list", "watch"]
+- apiGroups: ["batch"]
+  resources:
+  - jobs
+  verbs: ["get", "list", "watch"]
```

Comment:
Did I miss adding a test here?

Reply:
This is just documentation.  Thank you for adding this here too though!

Can you place add this to the manifests under `beats/deploy/kubernetes`, which is actually where needed?

Reply:
Ah! I was grepping in the `metricbeat` dir, not the parent dir, so missed these. Thanks.

Fixed in 81d1a33b7c.

Reply:
👍🏼